### ISSUE TITLE
[Bug]: Cant Implement in Dev Env [Fabric - 1.20.x]

### ISSUE THREAD

**Issue by xanthian** [terrarium-earth/Botarium#23]
### Bug Description

Mapping target name conflicts detected:
  METHODs earth/terrarium/botarium/common/fluid/impl/SimpleFluidContainer/[clear, net/minecraft/class_3829/method_5448]()V -> clear

### How to Reproduce?

add the following in build.gradle :

repositories {
    maven { url "https://maven.resourcefulbees.com/repository/maven-public/" }
    }
dependencies {
modApi "earth.terrarium:botarium-fabric-1.20:2.1.1"
}
update gradle and it fails with above error.

### Version

any 1.20.x 

### Mod Loader Version

1.20.1 - 0.14.21

### Mod Loader

Fabric

### Logs or additional context

link to my mod
https://github.com/xanthian/variantcraftingtables/tree/1.20

tried adding Botarium from Cursemaven also in case it was a maven related issue.

Im using Yarn mappings.
I dont actually want Botarium, but to add Ad Astra support I need it :(

### Code of Conduct

- [X] I have searched the issue tracker and confirmed that the issue has not been reported
- [X] I have checked the FAQ (if one is present) and my issue is not listed
- [X] I have verified that I am using the latest version of the mod
- [X] I have verified that I have installed all the required dependencies for the mod
- [X] I have verified that I do not have Optifine installed. We will close issues if we detect Optifine is in your pack. We cannot fix Optifine issues. Please do not report issues with Optifine present.

**Comment by xanthian** [comment]
Also, If i put the botarium mod in the run/mods folder it produces the same error.

**Comment by CodexAdrian** [comment]
This issue wont be fixable until 1.20.2 as it requires a breaking change to my api. You can use official mappings to get around this for now

**Comment by xanthian** [comment]
Any change on status on this?

**Comment by CodexAdrian** [comment]
1.20.2 version isnt released yet as Botarium will support Neoforge and there are still breaking changes being made to the capabilities and transfer systems

**Comment by xanthian** [comment]
Fixed it myself, remapped the method at fault using tinyremapper

```
tiny	2	0	intermediary	named
c	earth/terrarium/botarium/common/fluid/impl/SimpleFluidContainer	earth/terrarium/botarium/common/fluid/impl/SimpleFluidContainer
	m	()V	clear	clear$fixConflict
```
It doesn't need to break your API.

**Comment by CodexAdrian** [comment]
Fixed in 1.20.4
